run-python3.js

```diff
@@ -25,4 +25,5 @@ runPython3(...process.argv.slice(2)).catch(err => {
   if (!error.startsWith("Error: Command failed")) {
     console.error(error);
   }
+  return 1;
```

Comment:
this doesn't actually return exit code `1` no? I thin you need `process.exit(1)`

Reply:
oh good point, will fix that